src/java.base/share/classes/java/net/URLClassLoader.java

```diff
@@ -113,8 +120,15 @@ public URLClassLoader(URL[] urls) {
      * factory argument will be used as the stream handler factory to
      * obtain protocol handlers when creating new jar URLs.
      *
+     * @apiNote If the {@code parent} is specified as {@code null} (for the
```

Comment:
I think "If {@code parent}" is more correct, since you are implicitly referring to the parameter.